org.code.toboggan.filesystem/src/org/code/toboggan/filesystem/extensionpoints/file/IFSFileMoveExt.java

```diff
@@ -6,6 +6,10 @@
 import org.eclipse.core.resources.IFile;
 
 public interface IFSFileMoveExt extends ICoreExtension {
+	public void fileMoved(long fileID, IFile iFile, Path newFileLocation);
+	public void moveFailed(long fileId, IFile iFile, Path newFileLocation);
 	public void moveUndone(long fileID, IFile iFile, Path originalFileLocation, Path undoneFileLocation);
 	public void undoFailed(long fileID, IFile iFile, Path oldFileLocation, Path newFileLocation);
```

Comment:
Could we change this to `moveUndoFailed` instead of just `undoFailed`? The other interfaces also have `undoFailed` functions that mess with this.